python/pyproject.toml

```diff
@@ -26,10 +26,15 @@ dependencies = [
    "cffi",
    "numpy",
 ]
-optional-dependencies.ase = ["ase"]
-optional-dependencies.qcschema = ["qcelemental"]
-optional-dependencies.pyscf = ["pyscf"]
-optional-dependencies.test = [
+
+[project.optional-dependencies]
+ase = [
+  "ase==3.22.1; python_version < '3.8'",
```

Comment:
What is the motivation for having this tight constraint only with Python 3.7?

Reply:
Changed to ase<3.23. Version 3.23+ does not support Python 3.7 anymore and throws internal errors.